DependencyInjection/BazingaHateoasExtension.php

```diff
@@ -60,5 +61,13 @@ public function load(array $configs, ContainerBuilder $container): void
             ->getDefinition('hateoas.event_listener.xml')
             ->setPublic(true)
             ->replaceArgument(0, new Reference($config['serializer']['xml']));
+
+        if (PHP_VERSION_ID >= 80100 && class_exists(AttributeReader::class)) {
+            $container->register('hateoas.metadata.annotation_and_attributes_reader', AttributeReader::class)
+                ->setArgument(0, new Reference('annotation_reader'));
```

Comment:
The service "annotation_reader" has been removed as of `symfony/framework-bundle:^7.0`

Reply:
I've refactored the way the new `AttributeDriver` is configured in the bundle along with the annotation driver.

For this issue in particular, I've created a new private service `hateoas.configuration.metadata.annotation_reader` to provide `Doctrine\Common\Annotations\AnnotationReader` as a short-term solution.

https://github.com/willdurand/BazingaHateoasBundle/pull/104/files#diff-b542b053bcd2c59c85433c7f6e6f2f2c46aa452e7f15b6f315f668cb428bd400R87

Since `willdurand/hateoas` requires `doctrine/annotations` as a root package, it should be safe to create the service "hateoas.configuration.metadata.annotation_reader" definition.

Also, I would say the annotation driver should be marked as deprecated (I saw JMS did it), but I am not sure if you want to do the same.